src/i18n.c

```diff
@@ -0,0 +1,37 @@
+#include "i18n.h"
+#include <string.h>
+#include <stdlib.h>
+
+#define i18n_map_size 3
+char *i18n_map[i18n_map_size][4] = {
+    {"zh_CN", "激活 ", "转到“设置”以激活 ", "。"},
+    {"zh_TW", "啟用 ", "移至[設定]以啟用 ", "。"},
+    {"zh_HK", "啟用 ", "移至[設定]以啟用 ", "。"},
+};
+char *another_strcat(size_t len, char *strings[]){
+    unsigned long t = 0;
+    for (size_t i = 0; i < len; ++i)
+        t += strlen(strings[i]);
+    char *tmp = (char*)malloc((t+1) * sizeof(char));
```

Comment:
This malloc is never freed

Reply:
But the string needs to be passed to the drawer. It should behave like a literal string stored in the heap.

Reply:
`i18n` in activate_linux.c is not freed at the end of the program.

Reply:
So what's the better practice? Can I declare `i18n_info.title` as a char array with a finite length? Or is there any way to create something behaving the same as literal string?

Reply:
Just freeing it at the end of the program is good enough, I believe.